README.adoc

```diff
@@ -40,34 +61,58 @@ Our objective is to know if the Jenkins project published a new stable version,
 [cols="2a,2a"]
 |===
 |```
-## updatecli.yaml
-sources:
+## updatecli.yamltitle: Update Jenkins Version
+
+scms:
   default:
+    kind: github
+    spec:
+      user: olblak
+      email: me@olblak.com
+      owner: "olblak"
+      repository: "charts"
+      token: mySecretTokenWhichShouldNeverUsedThisWay
+      username: olblak
+      branch: "main"
+
+sources:
+  jenkins:
+    name: Get latest Jenkins version
     kind: jenkins
     spec:
       release: weekly
+
 conditions:
   docker:
-    name: "Docker Image Published on Registry"
+    name: "Test if Docker Image Published on Registry"
+    sourceID: jenkins
     kind: dockerImage
     spec:
       image: "jenkins/jenkins"
+
 targets:
-  imageTag:
+  bumpJenkins:
     name: "jenkins/jenkins docker tag"
+    sourceID: jenkins
```

Comment:
Same as above, less is more.

Reply:
`sourceID` is mandatory. 
It doesn't try to be smart, which could be an improvement

Reply:
Are you sure? For me, one source = no need to specify it

Last time (of many) I've done like that: https://github.com/jenkins-infra/docker-helmfile/blob/main/updatecli/updatecli.d/doctl.yml

Reply:
Interesting, that was my understanding as well but I tried this morning and it didn't work but maybe I did something wrong

Reply:
- https://github.com/updatecli/updatecli/blob/main/pkg/core/config/main.go#L206-L217 shows us that, for conditions, when the attribute `disablesourceinput` isn't specified, or set to `false`, then updatecli uses the only source available (or errors if there are multiple sources).
- https://github.com/updatecli/updatecli/blob/main/pkg/core/config/main.go#L244-L253 shows the same behavior for targets.

Reply:
That's true, it works, I was confused by the message `No architecture specified for the image "registry-1.docker.io/jenkins/jenkins:".`

from

```
WARNING: No architecture specified for the image "registry-1.docker.io/jenkins/jenkins:". Using the default "amd64".
✔ The Docker image registry-1.docker.io/jenkins/jenkins:2.335 exists and is available.

```